src/common-principles.md

```diff
@@ -309,6 +309,28 @@ field in `dataset_description.json` of each subdirectory of `derivatives` to:
 }
 ```
 
+!!! danger "Caution"
+
+    Sharing source data may help amend errors and missing data discovered
+    only with the reuse of the raw dataset in practice.
+    Therefore, from an Open Science perspective, it is RECOMMENDED to share
+    the source data whenever it is possible.
+
+    However, more stringent sharing limitations may apply to the source data
+    than those applicable to the raw data.
+    For example, human data almost always requires anonymization
+    before they can be redistributed,
+    or the subjects' consent form did not explicitly state that the source files
```

Comment:
I would use `pseudonymization` or `deidentification` instead of `anonymization`, as `anonymization` in the sense used in the GDPR is not really possible, unless data lose most of their scientific interest.

Reply:
See #1782 to apply to the whole repo.

Reply:
good point

What is the technical difference between `pseudonymization` or `deidentification`?

Reply:
No objections to change. I think I prefer deidentification in this case.

Reply:
I don't have access to a normative document such as the above ISO standards, but I see **deidentification** as the more general term which includes **anonymisation** and **pseudonymisation**.

**Pseudonymisation** merely substitutes directly identifying information. For example, names might be replaced by hashes and MRI images might be defaced. However, it remains possible to identify the person behind the personal data, often using additional information.

A quick [Google search](pseudonymisation vs anonymisation ISO standard) comes up  with[Anonymisation vs Pseudonymisation](https://hoggo.io/anonymisation-vs-pseudonymisation/) and other articles. It would be great if someone with access to ISO standards could show us the original wording and recommendations in these standards.

Reply:
I also favor "deidentification" as the term used.

Reply:
+1 for deidentification, and applying it to the whole spec, as suggested in 

- https://github.com/bids-standard/bids-specification/issues/1782